indexing-service/src/main/java/io/druid/indexing/common/task/IndexReprocessTask.java

```diff
@@ -45,29 +45,30 @@
 
 /**
  */
-public class VersionConverterTask extends AbstractFixedIntervalTask
+public class IndexReprocessTask extends AbstractFixedIntervalTask
 {
-  private static final String TYPE = "version_converter";
+  private static final String TYPE = "index_reprocess";
   private static final Integer CURR_VERSION_INTEGER = IndexIO.CURRENT_VERSION_ID;
 
-  private static final Logger log = new Logger(VersionConverterTask.class);
+  private static final Logger log = new Logger(IndexReprocessTask.class);
 
   @JsonIgnore
   private final DataSegment segment;
   private final IndexSpec indexSpec;
+  private final boolean force;
 
-  public static VersionConverterTask create(String dataSource, Interval interval)
+  public static IndexReprocessTask create(String dataSource, Interval interval, IndexSpec indexSpec, boolean force)
```

Comment:
maybe `SegmentConverterTask` is more appropriate? "reprocess" makes it sound like we might be re-processing the data.